Assets/Scripts/Enemy/EnemyFactory.cs

```diff
@@ -104,5 +104,13 @@ void AddTurretToRoster()
 
         // Define enemyInfo for each type.
         _enemyInfo.Add(EnemyType.TURRET, new EnemyInfo(EnemyMovements.TurretMovement, Globals.DirectTargeting, EnemyMovements.TurretSetup, bulletBehaviour, fireRate));
+        
+        _enemyPostSetups.Add(EnemyVariantType.SET, CreateSetTurretVariant);
+    }
+
+    void CreateSetTurretVariant(EnemyBehaviour eb)
```

Comment:
Are we explicitly saying that Set Targetting is only for Turrets ? I would probably 
```suggestion
    void CreateSetDirectionVariant(EnemyBehaviour eb)
```